subsys/shell/shell.c

```diff
@@ -570,6 +573,9 @@ void shell_init(const char *str)
 #ifdef CONFIG_TELNET_CONSOLE
 	telnet_register_input(&avail_queue, &cmds_queue, completion);
 #endif
+#ifdef CONFIG_WS_CONSOLE
+	ws_register_input(&avail_queue, &cmds_queue, completion);
```

Comment:
@andrewboie : Here's what I mean by "an app needs to call each console driver explicitly".